frontend/src/components/Forms/Sheet/Sheet.tsx

```diff
@@ -1,15 +1,31 @@
 import classes from "./Sheet.module.scss";
 import { X } from "lucide-react";
+import classNames from "classnames";
 
 interface SheetProps {
   hasCloseButton?: () => void;
   children?: React.ReactNode;
+  sheetLogin?: boolean;
```

Comment:
i feel like these names could be abbreviated to just 'login', 'gamemode', and 'leaderboard'